docs/Docker.md

```diff
@@ -45,5 +43,21 @@ docker run algorand/conduit init --importer algod --processors filter_processor
 With `conduit.yml` in your current working directory,
 launch the container:
 ```
-docker run -it -v $(pwd)/conduit.yml:/conduit/data/conduit.yml algorand/conduit
+docker run -it -v $(pwd)/conduit.yml:/data/conduit.yml algorand/conduit
+```
+
+# Mounting the Data Directory
+
+For production deployments, you should consider mounting the entire data directory. This way you can persist state across images during an upgrade, or for backups. The data directory is located at `/data`. When mounting a data directory, it must contain the `conduit.yml` file.
+
+```
+docker run -it -v $(pwd)/local_data_dir:/data algorand/conduit
 ```
+
+## Volume Permissions
+
+The container executes in the context of the `algorand` user with UID=999 and GID=999 which is handled differently depending on your operating system or deployment platform. During startup the container temporarily runs as root in order to modify the permissions of `/data`. It then changes to the `algorand` user. This can sometimes cause problems, for example if your deployment platform doesn't allow containers to run as the root user.
+
+### Use specific UID and GID
+
+On the host system, ensure the directory being mounted uses UID=999 and GID=999. If the directory already has these permissions you may override the default user with `-u 999:999`.
```

Comment:
Why would I want to override the default user with 999:999 if it already has those permissions?

Reply:
If you don't want the container to start as the root user. Added a clarifying comment.